packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -496,23 +592,27 @@ add (const Scalar& alpha,
      const Teuchos::RCP<Teuchos::ParameterList>& params)
 {
   using Teuchos::RCP;
+  using Teuchos::rcp;
   using Teuchos::rcpFromRef;
   using Teuchos::rcp_implicit_cast;
   using Teuchos::rcp_dynamic_cast;
-
+  using Teuchos::TimeMonitor;
   typedef Scalar                            SC;
   typedef LocalOrdinal                      LO;
   typedef GlobalOrdinal                     GO;
   typedef Node                              NO;
-  typedef RowMatrix<SC,LO,GO,NO>            row_matrix_type;
   typedef CrsMatrix<SC,LO,GO,NO>            crs_matrix_type;
+  typedef Map<LO,GO,NO>                     map_type;
   typedef RowMatrixTransposer<SC,LO,GO,NO>  transposer_type;
-
-  const std::string prefix = "TpetraExt::MatrixMatrix::add(): ";
-
+  typedef Import<LO,GO,NO>                  import_type;
+  typedef Export<LO,GO,NO>                  export_type;
+  typedef AddKernels<SC,LO,GO,NO>           AddKern;
+  std::string prefix_mmm = "TpetraExt::MatrixMatrix::add(): ";
```

Comment:
@mhoemmen I did make this change here, but otherwise the std::string constructor time is negligible. On cee-compute (1 thread) 0.04 s was spent in std::string::string out of 1500 s total CPU time. This was with MMM_Timings off, but I assume users would have that off if they really care about performance.